bindings/web5_uniffi_wrapper/src/credentials/presentation_definition.rs

```diff
@@ -1,4 +1,5 @@
 use crate::errors::Result;
+use futures::executor::block_on;
```

Comment:
why `futures::executor::block_on` vs alternatives like `tokio::task::spawn_blocking`?

Reply:
`futures` has a slightly more streamlined error handling, whereas with tokio I would have to add a mapper into our error type, see the `unwrap` below

```rust
    pub fn resolve(uri: &str) -> Self {
        let inner_result = task::spawn_blocking(move || {
            tokio::runtime::Handle::current().block_on(InnerResolutionResult::resolve(uri))
        }).unwrap(); // Handle unwrap or any error handling as required.

        Self(inner_result)
    }
```

Reply:
Weird, I guess uniffi requires tokio? I could've sworn I tested with futures, but oh well, guess I have to go the tokio route